concise-swid-tag.cddl

```diff
@@ -12,16 +12,14 @@ concise-swid-tag = {
   entity => entity-entry / [ 2* entity-entry ],
   ? link => link-entry / [ 2* link-entry ],
   ? payload-or-evidence,
-  global-attributes,
   * $$coswid-extension,
+  global-attributes,
 }
 
 payload-or-evidence //= ( payload => payload-entry )
-payload-or-evidence //= ( payload => [ 2* payload-entry ] )
```

Comment:
Is this an unrelated change?

Reply:
yes, that is a change that leaked over from the "isomorphic issue"